Library/Homebrew/github_packages.rb

```diff
@@ -58,6 +58,8 @@ def upload_bottles(bottles_hash, dry_run:)
       HOMEBREW_PREFIX/"bin/skopeo",
     ].compact.first
     unless skopeo.exist?
+      odie "no `skoepeo` and HOMEBREW_FORCE_HOMEBREW_ON_LINUX is set!" if Homebrew::EnvConfig.force_homebrew_on_linux?
```

Comment:
Probably there's a typo here:
```suggestion
      odie "no `skopeo` and HOMEBREW_FORCE_HOMEBREW_ON_LINUX is set!" if Homebrew::EnvConfig.force_homebrew_on_linux?
```

Reply:
Good catch, thanks @bayandin! addressing in https://github.com/Homebrew/brew/pull/11034